src/Windows/Avalonia.Win32/PopupImpl.cs

```diff
@@ -84,6 +83,10 @@ protected override IntPtr WndProc(IntPtr hWnd, uint msg, IntPtr wParam, IntPtr l
                     _maxAutoSize = null;
                     goto default;
                 case UnmanagedMethods.WindowsMessage.WM_MOUSEACTIVATE:
+                    if (_parent?.Handle is not null)
+                    {
+                        UnmanagedMethods.SetFocus(_parent.Handle.Handle);
```

Comment:
@maxkatz6 found an issue with focus management. I'm not sure if this is the correct way to handle it but if I don't set the focus to the parent when the popup is clicked, I'm not able to type in the popup if there's a textbox, for example.